src/fsharp/CompilerImports.fsi

```diff
@@ -25,13 +25,13 @@ open FSharp.Compiler.ExtensionTyping
 #endif
 
 /// This exception is an old-style way of reporting a diagnostic
-exception AssemblyNotResolved of (*originalName*) string * range
+exception AssemblyNotResolved of string * range (*originalName*)
```

Comment:
@nojaf Have you seen changes related to comments like this and ones below?

Reply:
Nope, I didn't really review the changes in this PR.